Module/tide_utils2.py

```diff
@@ -0,0 +1,193 @@
+import os
+import glob
+import numpy as np
+import pandas as pd
+import xarray as xr
+import matplotlib.pyplot as plt
+
+
+def load_Neah_Bay(datadir):
+    """
+    Function to load the Neah Bay tidal station data from 2015 - 2016
+    and returns a dataframe and a Datetime Index object
+    Datadir is the directory path to where the data is located
+    """
+    if not glob.glob(os.path.join(datadir, '*NeahBay.csv')):
+        return None
+    else:
+        NeahBay_2014 = pd.read_csv(datadir + "2014_NeahBay.csv",
+                                   parse_dates=['Date Time'],
+                                   index_col=['Date Time'])
+        NeahBay_2015 = pd.read_csv(datadir + "2015_NeahBay.csv",
+                                   parse_dates=['Date Time'],
+                                   index_col=['Date Time'])
+        NeahBay_2016 = pd.read_csv(datadir + "2016_NeahBay.csv",
+                                   parse_dates=['Date Time'],
+                                   index_col=['Date Time'])
+        NeahBay = NeahBay_2014.append(NeahBay_2015)
+        NeahBay = NeahBay.append(NeahBay_2016)
+        NeahBay.index.rename('datetime', inplace=True)
+        return NeahBay
+
+
+def load_Port_Townsend(datadir):
+    """
+    Function to load the Port Townsend tidal station data from 2015
+    & 2016. Supply the directory to where the csv files with the
+    data are saved. Returns None if files are not located in
+    specified directory.
+    """
```

Comment:
This could be combined with the above function in a more general form by including the tidal set name as an input. That way you wouldn't have to repeat essentially the same code as three functions